lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -742,7 +742,9 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
     isColumnSorted(col: DataColumn, direction: string): boolean {
         if (col && direction) {
             const sorting = this.data.getSorting();
-            return sorting && sorting.key === col.key && sorting.direction.toLocaleLowerCase() === direction;
+            return sorting &&
```

Comment:
when condition becomes so long in orther to improve the redability would be better move them in a private method with a name that explain what we are checking